package.json

```diff
@@ -9,8 +9,10 @@
   },
   "exports": {
     "./web": "./bundles/web.bundle.min.js",
-    "./mjs": "./lib/mjs/index.js",
-    ".": "./lib/cjs/index.js"
+    ".": {
+      "require": "./lib/cjs/index.js",
```

Comment:
@asiaziola what do you think? I believe we've been here before and it was causing some fuckups for some of the web bundlers?

Reply:
we can quickly test it i guess, can you release a beta version with the change so I can run tests on bundlers?

Reply:
also - shouldn't we attach types to both of the versions?
so it will look sth like:
```
  "require": {
        "types": "./not-sure-whats-the-path-to-types",
        "default": "./lib/cjs/index.js"
      }
```

Reply:
and I would probably rename it from mjs to esm